### ISSUE TITLE
Misaligned number of integrations in integration chart thinggy

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis-ui#1135]
Looks something like this in Chrome and Firefox on my end:

![selection_301](https://user-images.githubusercontent.com/1306050/31991621-7b2bd734-b978-11e7-87a3-33f5596e2ddb.png)

I guess we still are few years away from CSS centring properly :)

**Comment by gashcrumb** [comment]
Yeah, that's been bugging me :-)  The whole text is supposed to go inside the circle, isn't it?
